include/CMakeLists.txt

```diff
@@ -20,6 +20,6 @@ macro( add_subdirectory_if_module_enabled _dir )
   endif()
 endmacro()
 
-add_subdirectory( Core )
-add_subdirectory( Filtering )
-add_subdirectory( Numerics )
+add_subdirectory_if_module_enabled( Core )
+add_subdirectory_if_module_enabled( Filtering )
+add_subdirectory_if_module_enabled( Numerics )
```

Comment:
These should still be `add_subdirectory` -- `Core`, `Filtering`, and `Numerics` are Groups as opposed to Modules.